px/lg/values.yaml

```diff
@@ -0,0 +1,20 @@
+region: __DEFINED_AS_ARG__
+apod:
+lg_image:
+
+registry:
+
+lg_config_path: px-info_communities/
+
+looking_glass:
+  lg:
+    proxy_port:
+    authenticate: false
+    subdomain: px
+  lgadmin:
+    authenticate: true
+    proxy_port:
+    subdomain: pxadmin
```

Comment:
Might be a good idea to have these default in line with the defaults of the bird deployment:
https://github.com/sapcc/helm-charts/blob/master/px/bird/values.yaml#L25-L33